DS-Session-3/priorityQueue/virtualCommandPrompt/VirtualCommandPrompt.java

```diff
@@ -0,0 +1,214 @@
+package virtualCommandPrompt;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * class to create a virtual command prompt
+ * @author Ishita_Mathur
+ *
+ */
+public class VirtualCommandPrompt
+{
+	static List<Directory> listOfDirectory = new ArrayList<Directory>();
+	static String path = ".\\";
+
+	/**
+	 * method to implement the virtual command prompt
+	 */
+	public static void virtualCommandPrompt()
+	{
+		Directory root = new Directory("R:", new Date());
+		listOfDirectory.add(root);
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("***Welcome to Virtual Command Prompt***");
+		String signature = "R:\\";
+		System.out.print(signature + ">");
+		Directory currentDirectory = root;
+
+		while(true)
+		{
+			String commandLine = scanner.nextLine();
+			String[] commandWords = commandLine.split(" ");
+
+			switch(commandWords[0])
+			{

```

Comment:
Comments should be there for hard coded values `commandWords[0] & commandWords[1]`.